packages/server/graphql/mutations/reflectTemplatePromptUpdateGroupColor.ts

```diff
@@ -0,0 +1,61 @@
+import {GraphQLID, GraphQLNonNull, GraphQLString} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import standardError from '../../utils/standardError'
+import ReflectTemplatePromptUpdateGroupColorPayload from '../types/ReflectTemplatePromptUpdateGroupColorPayload'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+
+const reflectTemplatePromptUpdateGroupColor = {
+  groupColor: 'Update the groupColor of a reflection prompt',
+  type: ReflectTemplatePromptUpdateGroupColorPayload,
+  args: {
+    promptId: {
+      type: new GraphQLNonNull(GraphQLID)
+    },
+    groupColor: {
+      type: new GraphQLNonNull(GraphQLString)
+    }
+  },
+  async resolve(_source, {promptId, groupColor}, {authToken, dataLoader, socketId: mutatorId}) {
+    const r = await getRethink()
+    const now = new Date()
+    const operationId = dataLoader.share()
+    const subOptions = {operationId, mutatorId}
+    const prompt = await r
+      .table('CustomPhaseItem')
+      .get(promptId)
+      .run()
+    const viewerId = getUserId(authToken)
+
+    // AUTH
+    if (!prompt || !isTeamMember(authToken, prompt.teamId) || !prompt.isActive) {
+      return standardError(new Error('Team not found'), {userId: viewerId})
+    }
+
+    // VALIDATION
+    const {teamId} = prompt
+
+    // RESOLUTION
+    await r
+      .table('CustomPhaseItem')
+      .get(promptId)
+      .update({
+        groupColor,
+        updatedAt: now
+      })
+      .run()
+
+    const data = {promptId}
+    publish(
+      SubscriptionChannel.TEAM,
```

Comment:
looks like the color change isn't propagating to the other team members (ie when i change the color, my team members don't see this change unless they hit refresh). we can talk about how to wire that up later

Reply:
Oh that's interesting. Okay cool, would love to hear how that's wired up!